src/_data/site.json

```diff
@@ -6,8 +6,8 @@
         "frontenddash": "https://boards.greenhouse.io/flowfuse/jobs/4911532004"
     },
     "messaging": {
-        "tagLine": "Build bespoke, flexible, and resilient manufacturing applications",
-        "subtitle": "FlowFuse and Node-RED allows your engineers to deliver custom manufacturing applications that can grow to meet your changing requirements.",
+        "tagLine": "Elevate Node-RED ",
+        "subtitle": "Capture and leverage data. FlowFuse provides a complete platform to scale your production applications, increase developer velocity, and enhance security in order to accelerate innovation.",
         "title": "Bespoke, flexible, and resilient manufacturing applications"
```

Comment:
Why keep this the same @Hasmin-AC ?

Reply:
for title we should have something SEO friendly. Will update that with something better which incorporate the top search terms including node-red